### PR TITLE
Updates for Android 14 QPR1

### PR DESCRIPTION
**Pull request by grote**

### CODE REVIEW

File: build.plugins.toml [hunk 1]
```diff
@@ -5,9 +5,10 @@
 # 1.6.10 Android 13
 # 1.7.20 Android 13 (QPR2)
 # 1.8.10 Android 14
+# 1.9.0 Android 14 (QPR1)
```
Comment (chirayudesai) [inline build.plugins.toml]: QPR2

**Comment by chirayudesai** [comment]
Let's see what the CI tests say. Also worth doing a quick smoke test on device.

File: build.plugins.toml (hunk 1 above)
Comment (theimpulson) [reply build.plugins.toml]: Done

**Comment by chirayudesai** [comment]
Builds fine in AOSP locally.

**Comment by theimpulson** [comment]
Seems to build and work fine for me

**Review by chirayudesai — CHANGES_REQUESTED** [review]
Crashes with biometrics enabled https://zifb.in/raw/VSPqq9RC9F

**Review by chirayudesai — APPROVED** [review]

**Comment by chirayudesai** [comment]
> Crashes with biometrics enabled https://zifb.in/raw/VSPqq9RC9F

Seems to have solved itself afte reflash, likely a bad state temporarily.

**Pull request merged**